README.md

```diff
@@ -96,91 +96,79 @@ fn main() -> ! {
 }
 ```
 
+would output
+
 ``` console
 $ cargo run --bin hello
-Running `probe-run target/thumbv7em-none-eabi/debug/hello`
-flashing program ..
-DONE
-resetting device
-Hello, world!
-stack backtrace:
-0: 0x0000031e - __bkpt
-1: 0x000001d2 - hello::__cortex_m_rt_main
-2: 0x00000108 - main
-3: 0x000002fa - Reset
+    Finished dev [unoptimized + debuginfo] target(s) in 0.07s
+     Running `probe-run --chip nRF52840_xxAA target/thumbv7em-none-eabihf/debug/hello`
+  (HOST) INFO  flashing program (30.22 KiB)
+  (HOST) INFO  success!
+────────────────────────────────────────────────────────────────────────────────
+INFO:hello -- Hello, world!
+────────────────────────────────────────────────────────────────────────────────
+  (HOST) INFO  exiting because the device halted.
+To see the backtrace at the exit point repeat this run with
+`probe-run --chip nRF52840_xxAA target/thumbv7em-none-eabihf/debug/hello --force-backtrace`
```

Comment:
Do we really want to print this after every run? Seems like a lot of noise to me

Reply:
hrmm. I was trying to be helpful but i see your point– there are other config options but we don't print these either. dialed this down to `device halted without error` & squashed